src/libcore/atomic.rs

```diff
@@ -285,7 +286,8 @@ impl AtomicBool {
     ///
     /// let some_bool = AtomicBool::new(true);
     ///
-    /// let value = some_bool.store(false, Ordering::Relaxed);
+    /// assert_eq!(true, some_bool.compare_and_swap(false, false, Ordering::Relaxed));
```

Comment:
Could this also perhaps show off an example of a successful compare-and-swap? Also, our assertions are typically `assert_eq!(actual, expected)` instead of the other way around I believe.